### PR TITLE
New RN for Receipts mapurl

### PR DESCRIPTION
**Pull request by sarraloew**

### CODE REVIEW

File: src/tools-support/release-notes/api/2018_10_Platform.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+### Contents
+
+* [New MapUrl Field Added to Receipts v4 API](#v4receipts)
+
+### <a name="v4receipts"></a>New MapUrl Field Added to Receipts v4 API
+
+A new field, MapUrl, has been added to the Ground Transportation Receipt schema of the [Receipts v4](https://developer.concur.com/api-reference/receipts/get-started.html) API. This field contains a link to an image on Google Maps of the trip, if provided by the ground transportation supplier.
+#### What the User Sees
```
Comment (charlieconcur) [inline src/tools-support/release-notes/api/2018_10_Platform.md]: Need a line break both before and after section headings.

File: src/tools-support/release-notes/api/2018_10_Platform.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+layout: reference
+---
+
+### Contents
+
+* [New MapUrl Field Added to Receipts v4 API](#v4receipts)
+
+### <a name="v4receipts"></a>New MapUrl Field Added to Receipts v4 API
+
+A new field, MapUrl, has been added to the Ground Transportation Receipt schema of the [Receipts v4](https://developer.concur.com/api-reference/receipts/get-started.html) API. This field contains a link to an image on Google Maps of the trip, if provided by the ground transportation supplier.
```
Comment (charlieconcur) [inline src/tools-support/release-notes/api/2018_10_Platform.md]: The URI should be relative and let's make sure it uses the new versioned one (this redirects at present): /api-reference/receipts/v4.get-started.html

File: src/tools-support/release-notes/api/2018_10_Platform.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: SAP Concur Developer Center - Shared Platform API Release Notes, October 2018
+layout: reference
+---
+
+### Contents
```
Comment (charlieconcur) [inline src/tools-support/release-notes/api/2018_10_Platform.md]: Top level headings are H2 (`##`) -- take a look at the prior art at https://raw.githubusercontent.com/concur/developer.concur.com/preview/src/tools-support/release-notes/api/2018-09-30.md and let's make sure they match. We can pause and make a template for this as well if we want.

**Review by charlieconcur — CHANGES_REQUESTED** [review]
No underscores in file names or paths.
All lowercase for files or paths.

**Comment by sarraloew** [comment]
I've updated the file, do I need to add a comment at this level for visibility?

**Review by charlieconcur — APPROVED** [review]

**Comment by charlieconcur** [comment]
You don't need to leave a comment. The system will send an email with each commit AND I will check all pull requests every 24 hours or so.

**Pull request merged**